assets/js/modules/analytics/components/common/ExistingGTMPropertyError.js

```diff
@@ -0,0 +1,55 @@
+/**
+ * Analytics Existing GTM Property Error component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __, sprintf } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { STORE_NAME } from '../../datastore/constants';
+import { STORE_NAME as MODULES_TAGMANAGER } from '../../../tagmanager/datastore/constants';
+import ErrorText from '../../../../components/error-text';
+const { useSelect } = Data;
+
+export default function ExistingGTMPropertyError() {
+	const existingTag = useSelect( ( select ) => select( STORE_NAME ).getExistingTag() );
+	const gtmAnalyticsPropertyID = useSelect( ( select ) => select( MODULES_TAGMANAGER ).getSingleAnalyticsPropertyID() );
+	const gtmAnalyticsPropertyIDPermission = useSelect( ( select ) => select( STORE_NAME ).hasTagPermission( gtmAnalyticsPropertyID ) );
+
+	// Don't display error notice if:
+	if (
+		!! existingTag || // There is an existing tag.
```

Comment:
I think this needs to be removed after the recent change of making the two existing tags independent. I think that makes sense though, because `ExistingGTMPropertyError` should only be concerned with a GTM property and not a regular existing tag per its name. Whether it should be displayed together with the `ExistingTagError` or not (in our case it shouldn't) is more of a broader UI specificity that IMO shouldn't be handled inside this component.

Reply:
Updated.